hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -268,10 +268,13 @@ private void getSCMSignedCert(OzoneConfiguration config) {
 
       String pemEncodedCert = secureScmClient.getDataNodeCertificate(
           datanodeDetails.getProtoBufMessage(), getEncodedString(csr));
-      dnCertClient.storeCertificate(pemEncodedCert, true);
+      dnCertClient.storeCertificate(pemEncodedCert, true, false);
       datanodeDetails.setCertSerialId(getX509Certificate(pemEncodedCert).
           getSerialNumber().toString());
       persistDatanodeDetails(datanodeDetails);
+      // Get SCM CA certificate and store it in filesystem.
+      String pemEncodedRootCert = secureScmClient.getCACertificate();
```

Comment:
Should we get the CA certificate based on the DN certificate signed by CA? Does that container a signer certificate id?

Reply:
As of now we don't have functionality to look up certificates by subject or scm id. getCACertificate returns default certificate for SCM who signed it.